install/common-pre-install/compare_shcd_data-cpi.md

```diff
@@ -0,0 +1,78 @@
+# Compare the SHCD Data with CVT Inventory Data
+
+
+## 1. Generate paddle file using the canu validate tool
+
+1. Generate the paddle file.
+

```

Comment:
add https://github.com/Cray-HPE/docs-csm/blob/release/1.4/operations/network/management_network/validate_shcd.md link to generate paddle file

Reply:
https://github.com/abhishek-g-suresh/docs-csm/pull/5